internal/command/testdata/plan-vars-prompt/main.tf

```diff
@@ -0,0 +1,14 @@
+variable "resource_name" {
```

Comment:
I don't see how this test case exercises the expected static variable input functionality.